src/fe-gtk/maingui.c

```diff
@@ -1084,6 +1084,12 @@ mg_tab_close_cb (GtkWidget *dialog, gint arg1, session *sess)
 	}
 }
 
+static struct {
+    server *server;
+    char channel[CHANLEN];
+    char key[64];
+} _last_chan = { 0 };
```

Comment:
Yes, that is what I intended to do, as I wrote in the description of the pull req. I need some advice of where it should be stored.